velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -397,12 +443,19 @@ void AggregationFuzzer::go() {
           if (failed) {
             signatureWithStats.second.numFailed++;
           }
-        } else {
-          std::shared_ptr<ResultVerifier> customVerifier;
-          if (customVerification) {
-            customVerifier = customVerificationFunctions_.at(signature.name);
+        } else if (distinct) {
+          ++stats_.numDistinct;
```

Comment:
numDistinct -> numDistinctInputs

BTW, you need to introduce a separate count to not mix select distinct a, b, c with select array_agg(distinct x).

Reply:
Fixed and add `numDistinctAggregations`